src/objdump.py

```diff
@@ -142,7 +152,7 @@ class ArchSettings:
     re_includes_sp=re.compile(r"\b(sp|s8)\b"),
     sp_ref_insns=["addiu"],
     reloc_str="R_MIPS_",
-    objdump=["mips-linux-gnu-objdump", "-drz", "-m", "mips:4300"],
+    objdump=[f"{get_prefix()}-objdump", "-drz", "-m", "mips:4300"],
```

Comment:
This will be called even if MIPS_SETTINGS is not used, which is a problem if no MIPS toolchain is installed. Maybe there should be an objdump_cands property on ArchSettings which can contain all candidates for prefixes on each arch?

Reply:
Oh thats a good idea, let me do it